apps/frontpage/next.config.js

```diff
@@ -53,6 +53,11 @@ module.exports = {
         destination: '/docs/configure/telemetry',
         permanent: true,
       },
+      {
+        source: '/versions.json',
```

Comment:
`versions.json` was not a valid route name, so I had to configure this redirect. The query parameters are forwarded, so everything still works as it should.